File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/Practitioner.ts
import { Address } from "./Address";
import { Attachment } from "./Attachment";
import { Period } from "./Period";
import { CodeableConcept } from "./CodeableConcept";
import { ContactPoint } from "./ContactPoint";
import { HumanName } from "./HumanName";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";
import { BackboneElement } from "./BackboneElement";

export type PractitionerQualification = BackboneElement & {
    code: CodeableConcept;
    issuer?: Reference;
    period?: Period;
    identifier?: Identifier[];
};

export type Practitioner = DomainResource & {
    address?: Address[];
    name?: HumanName[];
    birthDate?: string;
    photo?: Attachment[];
    active?: boolean;
    communication?: CodeableConcept[];
    identifier?: Identifier[];
    qualification?: PractitionerQualification[];
    telecom?: ContactPoint[];
    gender?: string;
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Practitioner.py
from typing import Optional, List
from base import Address
from base import Attachment
from base import Period
from base import CodeableConcept
from base import ContactPoint
from base import HumanName
from base import DomainResource
from base import Reference
from base import Identifier
from base import BackboneElement

class Practitioner_Qualification(BackboneElement):
    code: CodeableConcept
    issuer: Optional[Reference] = None
    period: Optional[Period] = None
    identifier: Optional[List[Identifier]] = None

class Practitioner(DomainResource):
    address: Optional[List[Address]] = None
    name: Optional[List[HumanName]] = None
    birth_date: Optional[str] = None
    photo: Optional[List[Attachment]] = None
    active: Optional[bool] = None
    communication: Optional[List[CodeableConcept]] = None
    identifier: Optional[List[Identifier]] = None
    qualification: Optional[List[Practitioner_Qualification]] = None
    telecom: Optional[List[ContactPoint]] = None
    gender: Optional[str] = None